spec/paths/api@v1@finnSakerForPerson.yaml

```diff
@@ -0,0 +1,27 @@
+post:
+  summary: Hent saker med refusjon for gitt person
+  tags:
+    - Sak
+  #TODO skjul parameterblokken
+  requestBody:
+    content:
+      application/json:
+        schema:
+          type: object
+          required:
+            - personnummer
```

Comment:
personNummer?

Reply:
🤷‍♀ det er jo ett ord, personnummer, og camelcase betyr at man setter to ord sammen. 'personNummer' -> person nummer, som jeg tenker får en helt annen semantisk betydning (f.eks. 'person nummer to')

Men dette er jo veldig subjektivt 😄

Reply:
Rigtigt, det kan sagtens tolkes som 1 ord :) samme med fødselsnummer. Vi må bare blive enige om hva vi bruker. Jeg stemmer faktisk for å vi tolker dem som 1 år, dvs alt lowercase

Reply:
Ja enig, det viktigste er at vi er konsistente 👍